packages/webapp/src/elections/components/election-member-chips.tsx

```diff
@@ -72,7 +83,23 @@ interface DelegateChipProps {
     level?: string;
 }
 
-export const DelegateChip = ({ member, level }: DelegateChipProps) => {
+export const DelegateChip = ({ member, level }: DelegateChipProps) => (
+    <ElectionParticipantChip member={member} delegateLevel={level} isDelegate />
+);
+
+interface ElectionParticipantChipProps {
+    member: MemberData;
+    delegateLevel?: string;
+    isDelegate?: boolean;
+    electionVideoCid?: string;
+}
+
+export const ElectionParticipantChip = ({
+    member,
+    delegateLevel,
+    isDelegate = false,
```

Comment:
isDelegate is minimally used. Is there a future use for it? Or could you just default delegateLevel = 0 and use that as the flag in the code?

Reply:
Yeah, in fact, if we look at `GenericMemberChip` and `ElectionParticipantChip`, I hint with TODOs on that prop is likely to change or go away depending on the shape of the data coming in and whether we can simply rely on that. This was an easy, in-your-face, way to demonstrate the application of the delegate badge...but we'll likely get that from the member object when we hook it up.